frontend/src/modules/editor/contentObject/views/editorPageView.js

```diff
@@ -112,6 +112,7 @@ define(function(require){
       }, {
         success: _.bind(function(model, response, options) {
           var articleView = this.addArticleView(model);
+          articleView._skipRender = true; // prevent render of blocks in postRender
           articleView.addBlock();
         }, this),
```

Comment:
Does it make sense to remove that `addBlock` call completely, and just do this? 
(this seems to fix it for me, but I may have missed something)
```js
success: _.bind(function(model, response, options) {
  this.addArticleView(model);
}, this)
```